cocos/misc/camera-component.ts

```diff
@@ -170,6 +189,7 @@ export class Camera extends Component {
      * @zh 相机的渲染优先级，值越小越优先渲染。
      */
     @displayOrder(0)
+    @range([0, 65535, 1])
```

Comment:
So `priority` is unsigned short integer ?

Reply:
Yes, it should be

Reply:
Will this change break the compatibility ?
What happens if user's data has already been set to a float value or a negative value？